src/map/battle.c

```diff
@@ -7092,7 +7092,11 @@ static const struct battle_data {
 	{ "pet_status_support",                 &battle_config.pet_status_support,              0,      0,      1,              },
 	{ "pet_attack_support",                 &battle_config.pet_attack_support,              0,      0,      1,              },
 	{ "pet_damage_support",                 &battle_config.pet_damage_support,              0,      0,      1,              },
+#ifdef RENEWAL
+	{ "pet_support_min_friendly_re",        &battle_config.pet_support_min_friendly,        910,    0,      950,            },
```

Comment:
just curios, why a new config? this config seem redundant.
could have just remain one config, and assign different value based on `RE/PRE-RE`?

Reply:
Sure it's redundant. I just added a separate config for user friendliness, so that `pet_support_min_friendly` don't have to be adjusted if server is compiled in pre-RE mode.
This may gets removed, depending on `PET_INTIMACY_LOYAL` has changed to 910 or not. (See @hemagx' requested changes.)

Reply:
It hasn't changed. New config gets removed.